wrench/src/main.rs

```diff
@@ -0,0 +1,156 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+extern crate app_units;
+extern crate byteorder;
+extern crate bincode;
+extern crate webrender;
+extern crate glutin;
+extern crate gleam;
+extern crate webrender_traits;
+extern crate euclid;
+extern crate yaml_rust;
+extern crate image;
+#[macro_use]
+extern crate clap;
+#[macro_use]
+extern crate lazy_static;
+
+#[cfg(target_os = "windows")]
+extern crate dwrote;
+
+use glutin::{ElementState, VirtualKeyCode};
+use std::path::PathBuf;
+use webrender_traits::*;
+
+mod wrench;
+use wrench::{Wrench, WrenchThing};
```

Comment:
I think this is against rust style guidelines, where you are supposed to have all the `use` to follow all the `mod` things